pip_audit/_cache.py

```diff
@@ -46,6 +48,14 @@ def _get_pip_cache() -> Path:
     return http_cache_dir
 
 
+def _delete_legacy_cache_dir(current_cache_dir: Path, legacy_cache_dir: Path) -> None:
+    """
+    Deletes the legacy `pip-audit` if it exists.
+    """
+    if current_cache_dir != legacy_cache_dir:
+        shutil.rmtree(legacy_cache_dir)
```

Comment:
No need for a helper here; let's just inline this into `_get_cache_dir` IMO.

Reply:
The last commit now inline the deletion into `_get_cache_dir` :heavy_check_mark: